src/messages/MMDSCacheRejoin.h

```diff
@@ -97,6 +97,7 @@ class MMDSCacheRejoin final : public MMDSOp {
       first(f), alternate_name(altn), ino(pi), remote_ino(ri), remote_d_type(rdt), nonce(n), lock(l) {}
     bool is_primary() const { return ino > 0; }
     bool is_remote() const { return remote_ino > 0; }
+    bool is_referent() const { return remote_ino > 0; }
```

Comment:
just wanted to get clarification for **is_remote** and **is_referent** referring to the same data member i.e. `remote_ino`